website/docs/docs/collaborate/environments/what-are-environments.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "What are environments?"
```

Comment:
```suggestion
title: "Workspace environments in dbt"
```

Reply:
sugg:
- for consistency with the other article names / page names that are nearby this one. "question format" is uncommon in this area of the left sidebar

Reply:
went with `Environments in dbt`